modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -1304,6 +1304,142 @@ public void testAddObjectEntryWithDefaultLanguageId() throws Exception {
 				ObjectDefinitionConstants.SCOPE_SITE));
 	}
 
+	@Test
+	public void testAddObjectEntryWithDefaultValue() throws Exception {
+		ObjectDefinition objectDefinition = _createObjectDefinition(
+			Arrays.asList(
+				new BooleanObjectFieldBuilder(
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).localized(
+					true
+				).name(
+					"localizedBooleanObjectFieldName"
+				).build(),
+				new PicklistObjectFieldBuilder(
+				).indexed(
+					true
+				).labelMap(
+					LocalizedMapUtil.getLocalizedMap(
+						RandomTestUtil.randomString())
+				).listTypeDefinitionId(
+					listTypeDefinition.getListTypeDefinitionId()
+				).localized(
+					true
+				).name(
+					"localizedPicklistObjectFieldName"
+				).build()));
+
+		ObjectEntry objectEntry = _defaultObjectEntryManager.addObjectEntry(
+			dtoConverterContext, objectDefinition,
+			new ObjectEntry() {
+				{
+					properties = HashMapBuilder.<String, Object>put(
+						"localizedBooleanObjectFieldName", () -> null
+					).put(
+						"localizedPicklistObjectFieldName", () -> null
+					).build();
+				}
+			},
+			ObjectDefinitionConstants.SCOPE_COMPANY);
+
+		Map<String, Object> properties = objectEntry.getProperties();
+
+		Assert.assertNull(properties.get("localizedBooleanObjectFieldName"));
+		Assert.assertNull(properties.get("localizedPicklistObjectFieldName"));
```

Comment:
@pedro-oliveira446 can we use the existing `assertEquals` method here instead?

Reply:
I tried to do that, but it doesn't work. The test passes, but somehow, when I pass the properties like this:

`properties = HashMapBuilder.<String, Object>put(
							"localizedBooleanObjectFieldName_i18n", () -> null
						).put(
							"localizedPicklistObjectFieldName", () -> null
						).build();`
						
 they arrive as null [here](https://github.com/liferay/liferay-portal/blob/ece7b55a8e74de21ee6a190ad5029e65cfbc27fd/modules/apps/object/object-rest-test-util/src/main/java/com/liferay/object/rest/test/util/BaseObjectEntryManagerImplTestCase.java#L75), so it doesn’t actually test anything.